src/components/Feedback.tsx

```diff
@@ -201,14 +212,44 @@ export const Feedback: React.FC<Props> = ({ taxFormUserInput }: Props) => {
   }
   return (
     <div className="sdn-feedbackbar__container" id="sdn-feedbackbar-container">
-      <div className="govuk-grid-column-one-third">
-        <img src="/assets/images/icon-lock.svg" alt="" className="icon" />
-        <span>Bezpečné SSL pripojenie</span>
-      </div>
-      <div className="govuk-grid-column-one-third">
-        <img src="/assets/images/icon-correct.svg" alt="" className="icon" />
-        <span>Overené certifikovaným účtovníkom</span>
+      <div className="sdn-feedbackbar__useful">
+        {!isUsefulSubmitted ? (
+          <>
+            <span className="sdn-feedbackbar__useful-question">
+              Boli tieto informácie pre vás užitočné?
```

Comment:
Ja velmi nerozumiem aky ma takato otazka vyznam - toto je appka ktora vyplna danove priznanie, nie nejaky navod / informacia. Pytat sa to pocas flowu v appke mi pride zbytocne, lebo clovek este nevie ci to dotiahne do konca. Ak sa preklika az na koniec, tak je jasne ze to prenho uzitocne bolo :)

Mozno poslat mail po termine danoveho ze ci to zvladli vdaka nasej appke + navodu bez problemov?

Reply:
Ako hovorim vyssie, tak hlavny zmysel tohto PR je dat prec tie priblble badge a zjednotit to s navodmi, zevraj je to blocker pre Hanu

Su to urcite dobre pripomienky, ale na dlhsiu diskusiu s UX a copy, teraz sa mi nechce riesit taku prkotinu :D Idealne kebyze potom k tomu spravis task v Trelle

Reply:
Suhlas, nie je to nutne do v1.